packit.yaml

```
@@ -19,24 +19,13 @@ jobs:
   - job: copr_build
     trigger: pull_request
     targets:
-    - fedora-40
-    - fedora-41
-    - fedora-latest-aarch64
-    - fedora-development
     - centos-stream-9-x86_64
-    - centos-stream-9-aarch64
-    - centos-stream-10
 
   - job: tests
     trigger: pull_request
     targets:
-      - fedora-40
-      - fedora-41
-      - fedora-latest-aarch64
-      - fedora-development
-      - centos-stream-9-x86_64
-      - centos-stream-9-aarch64
-      - centos-stream-10
+      centos-stream-9-image-mode:
```

```suggestion
      centos-stream-9-x86_64:
```
could you please try this? The key here should be a valid build target

---

@lbarcziova Thanks for the hint! That indeed queued a test. I expect this to fail due to reasons not related to packit (https://issues.redhat.com/browse/TFT-3020). But it should at least boot the correct image.

I suppose I could do

     distros: ["CentOS-Stream-9-image-mode", "centos-stream-9-x86_64"]

to actually test on both classic rpm and bootc?

---

yes, the lists are supported, but if I remember correctly it should be without the architecture in the distros list, so 

```
distros: ["CentOS-Stream-9-image-mode", "centos-stream-9"]
```